document/core/syntax/instructions.rst

```diff
@@ -35,6 +35,16 @@ The following sections group instructions into a number of different categories.
 .. _syntax-fbinop:
 .. _syntax-ftestop:
 .. _syntax-frelop:
+.. _syntax-vunop:
+.. _syntax-vbinop:
+.. _syntax-vternop:
+.. _syntax-vtestop:
+.. _syntax-vshiftop:
+.. _syntax-viunop:
+.. _syntax-vibinop:
+.. _syntax-vsatbinop:
+.. _syntax-vfunop:
+.. _syntax-vfbinop:
```

Comment:
I think it's nicer if we put all the SIMD instructions into their own new section.

Reply:
Sg, added a new section: SIMD instructions.